command/init.go

```diff
@@ -798,15 +798,10 @@ Terraform has made some changes to the provider dependency selections recorded
 in the .terraform.lock.hcl file. Review those changes and commit them to your
 version control system if they represent changes you intended to make.`))
 		}
-	}
-
-	// TODO: Check whether newLocks is different from previousLocks and mention
-	// in the UI if so. We should emit a different message if previousLocks was
-	// empty, because that indicates we were creating a lock file for the first
-	// time and so we need to introduce the user to the idea of it.
```

Comment:
I think this TODO was out of date. Immediately above we already check if `newLocks` is different from `previousLocks` and output two different messages. The same applies to v0.14: https://github.com/hashicorp/terraform/blob/1c68b7b5818d8a592f0fd5955064a34907034095/command/init.go#L845-L872

Reply:
Agreed. I'm pretty sure I just saw the TODO, TODID it and then forgot to remove the comment afterwards.